.env

```diff
@@ -7,4 +7,4 @@ PORT=8000
 #Postgres authentication variables
 POSTGRES_DB=cantus_db
 POSTGRES_USER=cantus_admin
-#POSTGRES_PASSWORD=**PROVIDE PASSWORD HERE**
+POSTGRES_PASSWORD=monkeybutt
```

Comment:
an accidental part of the commit?? 😜😜

Reply:
I can edit this one out

Reply:
lol i knew i'd forget eventually. good catch